lib/phubme/slack.ex

```diff
@@ -5,21 +5,28 @@ defmodule PhubMe.Slack do
     Logger.error("[PhubMe][Error] " <> error_message)
   end
 
-  def send_private_message({_comment, [], _sender, _source}) do
+  def send_private_message(%IssueComment{nicknames: []}) do
     Logger.info("All procceed")
   end
 
-  def send_private_message({ comment, [nick_head | nick_tail], sender, source}) do
-    case Slack.Web.Chat.post_message(nick_head, formated_message(comment, sender, source), %{token: slack_token}) do
+  def send_private_message(%IssueComment{comment: comment,
+                                  nicknames: [nick_head | nick_tail],
+                                  sender: sender, source: source}=issue_comment) do
+    case Slack.Web.Chat.post_message(nick_head, formated_message(issue_comment), %{token: slack_token}) do
       %{"error" => "invalid_auth"} -> invalid_slack_auth_message
       %{"ok" => false} -> no_matching_channel_message(nick_head)
       _ -> matching_channel_message
     end
-    send_private_message({comment, nick_tail, sender, source})
+
+    send_private_message(%IssueComment{comment: comment,
+                                nicknames: nick_tail,
+                                sender: sender,
+                                source: source})
```

Comment:
Since you're updating only one field, you can use: `%{ issue_comment | nicknames: nick_tail }`.